foundation/docs/blog/posts/2023-review.md

```diff
@@ -0,0 +1,158 @@
+Friday hangout, and have various text channels for discussion. 
+
+[Join the PyTexas Discord](https://discord.gg/jNPAbcNukj){ .md-button .md-button--primary }
+
+### PyTexas Meetup
+
+After various discussions with Texas Pythonista's we discovered that Texas was missing
+a vital piece of its meetup ecosystem. Many local meetups had yet to make a return
+and many who did have a local meetup lived too far away to make it to the meetup
+consistently. For those who aren't aware, Texas is a _huge_ state. Major cities
+such as Dallas and Houston are large enough that traversing from one side to the 
+other can take multiple hours by car. In response, a few of the PyTexas Conference 
+organizers decided to start a PyTexas Virtual Meetup that is held once a month
+in our [Discord Server](https://discord.gg/jNPAbcNukj). We had our first meetup
+on September 12, 2023, and have settled into holding our Monthly Meetup on the 
+first Tuesday of the month at 8:00pm CST (except when there is a holiday conflict, in which
+case we will move to the second Tuesday).
+
+[Join the PyTexas Meetup](https://pytexas.org/meetup/join){ .md-button .md-button--primary }
+
+## 2024 and Beyond
+
+2022 was a recovery year, 2023 was a foundational building year, 2024 will _hopefully_
+be an expansion year.
+
+### The Goal
+
+The goal of the PyTexas Foundation is and will continue to be the stewardship
+of the Texas Python Community. Our goal is to not only put on our own events,
+but be able to informationally, instrumentally, and financially support other Python
+based events within the great state of Texas (and hopefully not just within the
+major cities).
+
+### Next Steps 
+
+#### **PyTexas 2024 Conference and Meetups**
+
+The Annual PyTexas 2024 Conference will be held April 19 - 21, 2024 in Austin,
+Texas at the Austin Central Public Library. This will be the first year since 2017
+where the conference will have tutorials the day before the main conference.
+Early Bird tickets are currently on sale [on our website](https://pytexas.org/2024)
+and the [CFP closes January 14, 2024](https://pretalx.com/pytexas-2024/).
+
+The PyTexas Meetup will continue to host its monthly meetup. If you'd like to
+speak at the meetup, fill out the form [on the meetup site homepage](https://www.pytexas.org/meetup/)
+
+#### **Secure More Funding**
+
+For nearly the entirety of its lifetime, the conference was the Foundation's
+only expense and fundraising event via sponsorships. With the expansion into 
+meetups and the eventual goal of supporting local meetups we have to rethink our 
+fundraising efforts. We hope to be able to roll out a sponsorship package for
+sponsoring the PyTexas Meetup and even the Foundation overall within this coming
+year, so keep an eye on our website and socials for its announcement.
+
+#### **Increase Awareness Through Marketing**
+This mostly means more social media awareness. We're going to experiment with
```

Comment:
Did you want a newline here? Running this through `prettier` might fixup all the little things